### PR TITLE
feat(docs): supply deps to new codesandboxes

### LINKED ISSUE #1273: docs: put all general dependencies in the "open in sandbox" experience
- click open in sandbox
- have glamor, ps...theme, etc all available in every example

### PR DESCRIPTION
**Pull request by danethurber**
resolves #1273

### CODE REVIEW

File: docs/src/components/code-block/actions.tsx [hunk 1]
```diff
@@ -8,6 +8,37 @@ import pkg from '../../../package.json'
 import usePrevious from '../../hooks/use-previous'
 import styles from './styles.module.css'
 
+const CODESANDBOX_CSS_FILE = `
```
Comment (danethurber) [inline docs/src/components/code-block/actions.tsx:11]: i could make this cleaner....maybe use file-loader or something....but this was easy...

File: docs/src/components/code-block/actions.tsx [hunk 2]
```diff
@@ -45,10 +76,19 @@ export const CodeSandboxAction: React.FC<CodeSandboxActionProps> = props => {
       example={props.code}
       examplePath="does/not/do/anything/but/is/required.tsx"
       dependencies={{
-        '@babel/runtime': 'latest'
+        '@pluralsight/ps-design-system-core': 'latest',
+        '@pluralsight/ps-design-system-featureflags': 'latest',
+        '@pluralsight/ps-design-system-normalize': 'latest',
+        '@pluralsight/ps-design-system-theme': 'latest',
+        '@babel/runtime': 'latest',
```
Comment (danethurber) [inline docs/src/components/code-block/actions.tsx:83]: did i miss any?

Comment (jaketrent) [inline docs/src/components/code-block/actions.tsx:83]: why's this one needed?

File: docs/src/components/code-block/actions.tsx [hunk 3]
```diff
@@ -8,6 +8,37 @@ import pkg from '../../../package.json'
...
+const CODESANDBOX_INDEX_FILE = `
+import "@pluralsight/ps-design-system-core";
+import "@pluralsight/ps-design-system-normalize";
+import "./styles.css"
+
+import Theme from '@pluralsight/ps-design-system-theme'
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import Example from './example';
```
Comment (jaketrent) [inline docs/src/components/code-block/actions.tsx:26]: Magical

File: docs/src/components/code-block/actions.tsx [hunk 4]
```diff
@@ -8,6 +8,37 @@ import pkg from '../../../package.json'
...
+const CODESANDBOX_CSS_FILE = `
+html,
+body {
+  background: var(--psColorsBackgroundDark2);
+}
+`
+const CODESANDBOX_INDEX_FILE = `
+import "@pluralsight/ps-design-system-core";
+import "@pluralsight/ps-design-system-normalize";
+import "./styles.css"
```
Comment (jaketrent) [inline docs/src/components/code-block/actions.tsx:20]: awesome!

**Review by jaketrent — APPROVED** [review]

File: docs/src/components/code-block/actions.tsx (hunk 2 above)
Comment (danethurber) [reply docs/src/components/code-block/actions.tsx:83]: babel runtime was there previously. codesandbox uses it for typescripty stuff i think

**Pull request merged**